apmpackage/apm/manifest.yml

```diff
@@ -146,6 +146,98 @@ policy_templates:
             required: false
             show_user: false
             default: true
+          - name: max_header_bytes
+            type: integer
+            title: Maximum size of a request's header (bytes)
+            required: false
+            show_user: false
+            default: 1048576
+          - name: idle_timeout
+            type: text
+            title: Idle time before underlying connection is closed
+            required: false
+            show_user: false
+            default: "45s"
+          - name: read_timeout
+            type: text
+            title: Maximum duration for reading an entire request
+            required: false
+            show_user: false
+            default: "3600s"
+          - name: shutdown_timeout
+            type: text
+            title: Maximum duration before releasing resources when shutting down
+            required: false
+            show_user: false
+            default: "30s"
+          - name: write_timeout
+            type: text
+            title: Maximum duration for writing a response
+            required: false
+            show_user: false
+            default: "30s"
+          - name: max_connections
+            type: integer
+            title: Simultaneously accepted connections
+            description: 0 for unlimited
+            required: false
+            show_user: false
+            default: 0
+          - name: response_headers
+            type: yaml
+            title: Custom HTTP headers added to HTTP responses
+            description: Might be used for security policy compliance.
+            required: false
+            show_user: false
+          - name: expvar_enabled
+            type: bool
+            title: Enable APM Server Golang expvar support
+            description: exposed under /debug/vars
+            required: false
+            show_user: false
+            default: false
+          - name: tls_enabled
+            type: bool
+            title: Enable TLS
+            required: false
+            show_user: false
+            default: false
+          - name: tls_certificate
+            type: text
+            title: File path to server certificate
+            description: Required when TLS is enabled.
+            required: false
+            show_user: false
+          - name: tls_key
+            type: text
+            title: File path to server certificate key
+            description: Required when TLS is enabled.
+            required: false
+            show_user: false
+          - name: tls_key_passphrase
+            type: password
+            title: Passphrase for decrypting the certificate key
+            required: false
+            show_user: false
```

Comment:
@axw I will remove the `tls_key_passphrase` support for now, as it would still be stored in clear text. I prefer to not support it for now.

Reply:
SGTM